src/_deflate.c

```diff
@@ -263,8 +263,6 @@ static struct PyModuleDef deflate_module = {PyModuleDef_HEAD_INIT, "_deflate",
                                             deflate_methods};
 
 PyMODINIT_FUNC PyInit__deflate(void) {
-    Py_Initialize();
```

Comment:
This function is used to initialize a Python interpreter when embedding it in code. It is not supposed to be used in extension modules.

Reply:
Good to know! I probably copied it from somewhere when I started and just never questioned it.